docs/feature_flags.md

```diff
@@ -125,7 +125,40 @@ histogram (albeit via the text format). With this flag enabled, Prometheus will
 still ingest those conventional histograms that do not come with a
 corresponding native histogram. However, if a native histogram is present,
 Prometheus will ignore the corresponding conventional histogram, with the
-notable exception of exemplars, which are always ingested.
+notable exception of exemplars, which are always ingested. To keep the
+conventional histograms as well, enable `scrape_classic_histograms` in the
+scrape job.
+
+NOTE: Due to the protobuf serialization, conventional histogram `le` labels
```

Comment:
I believe the `NOTE:` will make this one paragraph render in a blue box, but all the following paragraphs will be normal again. Which is confusing, because it makes the following paragraph appear as unrelated to this one blue paragraph.

It's probably better to start with a line in italics like `_Note about the format of `le` and `quantile` label values:_`, followed by a blank line, and then all the text we already have without the `NOTE:`.